impl/client/src/test/javascript/prompting/PromptPanel.spec.js

```diff
@@ -1496,6 +1496,7 @@ define(["dojo/number", "dojo/i18n", "common-ui/prompting/PromptPanel",
       expect(state.parametersChanged).toBe(panel.parametersChanged);
       expect(state.autoSubmit).toBe(panel.autoSubmit);
       expect(state.page).toBe(paramDefn.page);
+      expect(state.isSuppressSubmit).toBe(panel.isSuppressSubmit);
```

Comment:
All fine here, but would it be possible to add tests for:
1. when a parameter widget changes its parameter, and its `needsUpdateOnNextRefresh` property is `true`, then the next prompt panel refresh happens with `isSuppressSubmit` as `true`
2. when `isSuppressSubmit` is true, a submit that would otherwise be made is effectively suppressed.
3. the state being observable, from within `onAfterUpdate`, with the values used by the update/refresh/init 
4. when during a refresh, a component that would not otherwise be updated, but which has its `needsUpdateOnNextRefresh` set to `true` is updated anyway.